src/Components/DapperScrollbars/DapperScrollbars.tsx

```diff
@@ -62,6 +62,9 @@ export const DapperScrollbars: FunctionComponent<DapperScrollbarsProps> = ({
   removeTrackYWhenNotUsed = true,
   removeTrackXWhenNotUsed = true,
 }) => {
+  const scrollEl = useRef<any>(null)
+  const [scrollTopPos, setScrollTopPos] = useState(scrollTop)
```

Comment:
The `scrollTop` & `scrollLeft` props set the initial state but can no longer affect the scroll position after mount. If we implement this internal state we'll also need to manually handle prop updates.

Reply:
Fixed.